### ISSUE TITLE
Remove the default observer of Earth for a Map without a defined observer?

### ISSUE THREAD

**Issue by ayshih** [sunpy/sunpy#3387]
(This is where to continue discussion from #3118 and other places.)

Currently, Maps without a defined observer will use a default observer of Earth.  (Curiously, this default doesn't rely on any default-observer behavior in the coordinates machinery, so the proposed change in #3386 isn't relevant.)  Should this be changed?

**Comment by wtbarnes** [comment]
I'm guessing this choice has not been very contentious given the lack of discussion here in the last 6.5 years.

What would be the proposed alternative? Setting `None`? In the past, Earth was a sensible default, but as the number of observatories off the Sun-Earth line grows, it may make less and less sense. Maybe it would be more feasible to set this to `None` and have a default assumption of Earth for individual sources if they are known to be missing their observer keywords?

**Comment by ayshih** [comment]
Yes.  It is already possible through private API to specify a source-specific default observer, so I'd remove the generic default observer.
